knox/models.py

```diff
@@ -38,5 +38,10 @@ class AuthToken(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     expiry = models.DateTimeField(null=True, blank=True)
 
+    class Meta:
+        # Work around for a bug in Django:
+        # https://code.djangoproject.com/ticket/19422
+        abstract = 'knox' not in settings.INSTALLED_APPS
```

Comment:
Reading the ticket, this seems to only apply to pre "app loading" times in Django. I don't think we care.

Reply:
Basically the problem is following, if end user want to inherit from knox.models.AuthToken then he will get `RuntimeError: Model class knox.models.AuthToken doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.`

Reply:
I still have the feeling that this is not relevant anymore, did you witness the problem on a recent / supported version of Django?

Reply:
I'm not sure this is what you mean, but I can confirm that on Django 2.2 this (or the lack of this) causes issues making knox less plug-able.
For example, when inheriting from AuthToken and adding field to the model, creating a multi-column constraint (unique_together) is not possible (there are two tables).

Reply:
yes, that the same issue that DRF auth token had, so I just copied their solution here